src/elements/flip-card/flip-card-summary/flip-card-summary.scss

```diff
@@ -42,6 +42,9 @@
   padding-block: var(--sbb-spacing-responsive-s) var(--sbb-spacing-responsive-xs);
```

Comment:
Somehow the padding of the flipped state changed. On purpose?
https://lyne-components-visual-regression-diff-pr3001.app.sbb.ch/compare/sbb-flip-card/flipped

Reply:
I think the previous one was wrong 🤔

Reply:
Then it should be ok :)